apps/journeys-admin/src/components/ContainedIconButton/ContainedIconButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { ReactElement, ReactNode } from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import Card from '@mui/material/Card'
+import CardActionArea from '@mui/material/CardActionArea'
+import Stack from '@mui/material/Stack'
+import Icon from '@mui/material/Icon'
+import { ImageThumbnail } from '../ImageThumbnail'
+
+interface ContainedIconProps {
+  thumbnailIcon: ReactNode
+  actionIcon: ReactNode
+  label: string
+  description?: string
+  imageSrc?: string
+  imageAlt?: string
+  loading?: boolean
+  onClick: () => void
+}
+
+export function ContainedIconButton({
+  thumbnailIcon,
+  actionIcon,
+  label,
+  description,
+  imageSrc,
+  imageAlt,
+  loading,
+  onClick
+}: ContainedIconProps): ReactElement {
+  return (
+    <Card variant="outlined" sx={{ borderRadius: 2 }}>
+      <CardActionArea onClick={onClick}>
+        <Stack direction="row" alignItems="center" spacing={3} sx={{ p: 2 }}>
+          <Box>
+            <ImageThumbnail
+              imageSrc={imageSrc}
+              imageAlt={imageAlt}
+              icon={thumbnailIcon}
+              loading={loading}
+            />
+          </Box>
+          <Box flexGrow={1} minWidth={0}>
+            <Typography
+              variant="subtitle2"
+              sx={{
+                textOverflow: 'ellipsis',
+                whiteSpace: 'nowrap',
+                overflow: 'hidden'
+              }}
+            >
+              {label}
+            </Typography>
+
+            {description !== undefined && (
```

Comment:
The check below will check both null and undefined https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality

```suggestion
            {description != null && (
```

Reply:
fixed in cc28faa3eac2c914812f6794c2f77c31032c2471